Library/Reachability.swift

```diff
@@ -19,7 +19,7 @@ public enum Reachability {
 private let reachabilityProperty: MutableProperty<Reachability> = {
   guard
     let networkReachability = networkReachability(),
-    reachabilityFlags = reachabilityFlags(forNetworkReachability: networkReachability)
+    let reachabilityFlags = reachabilityFlags(forNetworkReachability: networkReachability)
```

Comment:
paging @stephencelis would love your eyes on this file since you wrote all of this. i think it's all pretty straight forward tho.